### PR TITLE
Improve azure logs

### PR DESCRIPTION
**Pull request by mucsi96**

### CODE REVIEW

File: packages/w3c-webdriver/test-env/d.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@ declare module 'iedriver' {
   export const path: string;
```
Comment (spnraju) [inline packages/w3c-webdriver/test-env/d.ts]: Any specific reason we named this file as d?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -54,14 +61,14 @@ jobs:
     name: Internet_Explorer
     vmImage: 'vs2017-win2016'
     nodeVersion: '12.x'
-    testCommand: 'npm run test:internet-explorer -- -- --ci'
+    browser: 'internet-explorer'
```
Comment (spnraju) [inline azure-pipelines.yml:64]: Do you think we can remove this browser from the list?

**Review by spnraju — APPROVED** [review]

File: azure-pipelines.yml (hunk 2 above)
Comment (mucsi96) [reply azure-pipelines.yml:64]: I thinks this still can be beneficial for someone. Still many developers have to support IE. Maybe next year if Edge became more stable.

File: packages/w3c-webdriver/test-env/d.ts (hunk 1 above)
Comment (mucsi96) [reply packages/w3c-webdriver/test-env/d.ts]: renamed to `declarations.ts`

**Pull request merged**